spellbook/models/seaport/ethereum/seaport_ethereum_transfers.sql

```diff
@@ -0,0 +1,822 @@
+          left join tokens_ethereum.nft n on n.contract_address = concat('0x',substr(a.nft_address,3,40))
+          left join tokens_ethereum.erc20 t1 on t1.contract_address = case when 
+          concat('0x',substr(a.price_token,3,40)) = '0x0000000000000000000000000000000000000000' then 
+          '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+          else concat('0x',substr(a.price_token,3,40)) end                                
+          left join prices.usd p1 on p1.contract_address = case when 
+          concat('0x',substr(a.price_token,3,40)) = '0x0000000000000000000000000000000000000000' then 
+          '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+          else concat('0x',substr(a.price_token,3,40)) end
+          and p1.minute = date_trunc('minute', a.block_time)
+          and p1.blockchain = 'ethereum')
+                                  
+,p3_call as (select 'order' as main_type
+          ,call_tx_hash as tx_hash
+          ,call_block_time as block_time
+          ,call_block_number as block_number
+          ,max(get_json_object(get_json_object(order, "$.parameters"), "$.orderType")) as order_type_id
+      from {{ source('seaport_ethereum','Seaport_call_fulfillOrder') }}
+      {% if is_incremental() %} -- this filter will only be applied on an incremental run 
+      where call_block_time >= (select max(block_time) from {{ this }}) 
+      {% endif %}
+     group by 1,2,3,4
+     union all
+    select 'advanced_order' as main_type
+          ,call_tx_hash as tx_hash
+          ,call_block_time as block_time
+          ,call_block_number as block_number
+          ,max(get_json_object(get_json_object(advancedOrder, "$.parameters"), "$.orderType")) as order_type_id
+      from {{ source('seaport_ethereum','Seaport_call_fulfillAdvancedOrder') }} 
+      {% if is_incremental() %} -- this filter will only be applied on an incremental run 
+      where call_block_time >= (select max(block_time) from {{ this }}) 
+      {% endif %}
+      group by 1,2,3,4)
+ 
+,p3_evt as (select c.main_type
+          ,c.tx_hash
+          ,c.block_time
+          ,c.block_number
+          ,c.order_type_id
+          ,'offer' as sub_type
+          ,offer_idx as sub_idx
+          ,e.offerer as sender
+          ,e.recipient as receiver
+          ,e.zone
+          ,concat('0x',substr(get_json_object(offer2, "$.token"),3,40)) as token_contract_address 
+          ,get_json_object(offer2, "$.amount") as original_amount
+          ,get_json_object(offer2, "$.itemType") as item_type
+          ,get_json_object(offer2, "$.identifier") as token_id
+          ,e.contract_address as exchange_contract_address
+          ,e.evt_index
+          from 
+          (select *, posexplode(offer) as (offer_idx, offer2) from {{ source('seaport_ethereum','Seaport_evt_OrderFulfilled') }}) e 
+          inner join p3_call c on c.tx_hash = e.evt_tx_hash
+          {% if is_incremental() %} -- this filter will only be applied on an incremental run 
```

Comment:
similar to p1_evt above, this logic should be applied to 'e' alias and probably moved to the subquery itself on line 453